test/test_transforms.py

```diff
@@ -351,3 +348,41 @@ def test_batched_recursive_linear_transform():
     y = transform(x)
     assert y.shape == x.shape
     assert jnp.allclose(x, transform.inv(y), atol=1e-6)
+
+
+@pytest.mark.parametrize(
+    "constraint, shape",
+    [
+        (constraints.circular, (3,)),
+        (constraints.complex, (3,)),
+        (constraints.corr_cholesky, (10, 10)),
+        (constraints.corr_matrix, (21,)),
```

Comment:
@tillahoffmann This test is failing in my system with jax dev. Could we relax this to 15 (which passes)?

Reply:
Sounds good. I can send a mini PR or integrate it into #1538?

Reply:
Either of them sounds good to me. Thanks, Till!